TACC/Core-CMS-Custom
# feat: a2cps_cms

## Overview

Migrate project from Core-CMS-Resources.

## Related

- fixed since https://github.com/TACC/Core-CMS/pull/688

## Testing

1. Verify [build](https://github.com/TACC/Core-CMS-Custom/actions/runs/5905138789/job/16018661425) and [deploy](https://jenkins01.tacc.utexas.edu/job/Core_Portal_Deploy/3073/console) succeed on https://pprd.a2cps.tacc.utexas.edu/.
2. Verify all CSS files load.
3. Verify new and deprecated templates load.
4. Verify new and deprecated templates work.
5. Verify https://pprd.a2cps.tacc.utexas.edu/ and https://a2cps.tacc.utexas.edu/ match (styles, not content).
	<sub>**Note**: Buttons are supposed to be blue now (example: contact form on [pprd](https://pprd.a2cps.tacc.utexas.edu/contact/) and [prod](https://a2cps.org/contact/)).</sub>

## UI

### CSS

| local | remote |
| - | - |
| <img width="1262" alt="a2cps local ui" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/5dd29a8d-d7d3-403c-9047-a30f6d3e2403"> | <img width="1262" alt="a2cps pprd ui" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/331966cc-7fb9-43d9-8a29-b47efcb26b26"> |

### Templates

| deprecated | new |
| - | - |
| <img width="1262" alt="a2cps deprecated template" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/2c25d3ff-e01c-4a56-9cc1-64720c6c666c"> | <img width="1262" alt="a2cps new template" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/5ed37dc0-93fd-47fb-b68b-61ffed205ff9"> |
